src/analysis/normed_space/compact_operator.lean

```diff
@@ -376,55 +371,54 @@ hf
 end continuous
 
 lemma is_closed_set_of_is_compact_operator {𝕜₁ 𝕜₂ : Type*} [nontrivially_normed_field 𝕜₁]
-  [nontrivially_normed_field 𝕜₂] {σ₁₂ : 𝕜₁ →+* 𝕜₂} [ring_hom_isometric σ₁₂] {M₁ M₂ : Type*}
-  [seminormed_add_comm_group M₁] [normed_add_comm_group M₂] [normed_space 𝕜₁ M₁]
-  [normed_space 𝕜₂ M₂] [complete_space M₂] :
+  [nontrivially_normed_field 𝕜₂] {σ₁₂ : 𝕜₁ →+* 𝕜₂} {M₁ M₂ : Type*} [seminormed_add_comm_group M₁]
```

Comment:
It hardly matters but can we get away with `normed_field` here?
```suggestion
  [normed_field 𝕜₂] {σ₁₂ : 𝕜₁ →+* 𝕜₂} {M₁ M₂ : Type*} [seminormed_add_comm_group M₁]
```

Reply:
(and I guess similarly for `compact_operator_topological_closure` and `is_compact_operator_of_tendsto` below)

Reply:
Indeed, there is no need for any restriction at the target. Thanks!